From 938e2a19dcc775e2934cb276896fba4851f419cf Mon Sep 17 00:00:00 2001
From: Robert Marshall <rmarshall@redhat.com>
Date: Wed, 21 Jan 2015 11:27:32 -0500
Subject: [PATCH 28/41] Update man page to include default config file for
 s390x bootloader.

The grubby man page defined the default config file for every
architecture's bootloader except s390. Added this information
to the man page.

Resolves bz#1158904

Conflicts:
	grubby.8
---
 grubby.8 | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/grubby.8 b/grubby.8
index 5ff486f..85e22c5 100644
--- a/grubby.8
+++ b/grubby.8
@@ -31,7 +31,8 @@ file is in \fB/boot/efi/EFI/redhat/grub.cfg\fR.  On Intel ia64 platforms,
 is \fB/boot/efi/EFI/redhat/elilo.conf\fR. On PowerPC platforms, systems based
 on Power8 now support \fBgrub2\fR as a bootloader and store using a default
 config stored in \fB/boot/grub2/grub.cfg\fR. The earlier Power7 systems use \fByaboot\fR
-parsing and the configuration file should be in \fB/etc/yaboot.conf\fR.
+parsing and the configuration file should be in \fB/etc/yaboot.conf\fR.  On
+s390 platforms the \fBzipl bootloader\fR will read from \fB/etc/zipl.conf\fR.
 
 There are a number of ways to specify the kernel used for \fB-\-info\fR,
 \fB-\-remove-kernel\fR, and \fB-\-update-kernel\fR. Specificying \fBDEFAULT\fR
-- 
2.4.3

